framework/include/media/MediaRecorderObserverInterface.h

```diff
@@ -29,11 +29,14 @@
 #ifndef __MEDIA_MEDIARECOREROBSERVERINTERFACE_H
 #define __MEDIA_MEDIARECOREROBSERVERINTERFACE_H
 
-#include <media/MediaRecorder.h>
+//#include <media/MediaRecorder.h>
```

Comment:
How about deleting this or marking ToDo?

Reply:
i will fix it.